qtopconf.yaml

```diff
@@ -1,9 +1,11 @@
 # Default configuration file for qtop.
-# IMPORTANT: a blank line MUST be left after every line of python code.
+# IMPORTANT: a blank line MUST be left after every line of *python code*.
 
 # scheduler possible values (currently supported): pbs, oar, sge
 # auto will try to detect available batch commands in the current system
+# env variable QTOP_SCHEDULER should be unset if QTOPCONF_YAML is set to auto
```

Comment:
in the old scheme of things, this was a bad side-product of the design. Now there's no need for that, so I'm removing the comment